hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -285,46 +285,48 @@ sudo reboot
 ```
 
 ## USB mass storage boot
-This is only available in the BETA release and requires updated (not yet released) firmware via [rpi-update](../../raspbian/applications/rpi-update.md). If you aren't already familiar with how to use a USB drive for the root filesystem then you probably want to wait until this feature is in the default Raspbian image.
+This is only available in the BETA release and requires updated firmware via [rpi-update](../../raspbian/applications/rpi-update.md). If you aren't already familiar with how to use a USB drive for the root filesystem then you probably want to wait until this feature is in the default Raspbian image.
 
-There's no support for migrating a SD card image to a USB drive. It's possible but the process can potentially be quite involved and varies according to your original setup. Please see [STICKY: HOWTO: Move the filesystem to a USB stick/Drive](https://www.raspberrypi.org/forums/viewtopic.php?f=29&t=44177&start=350)
+There's no support for migrating a SD card image to a USB drive. It's possible but the process can potentially be quite involved and varies according to your original setup. Please see [STICKY: HOWTO: Move the filesystem to a USB stick/Drive](https://www.raspberrypi.org/forums/viewtopic.php?f=29&t=44177&start=350).
 
 ## BETA setup instructions
 These instructions assume that you are familiar with manual firmware and bootloader updates and understand how to revert to a previous version if you want to revert the changes. If not, please wait until the features are available in a full Raspbian release image.
 
-There will be a post on the Raspberry Pi [General Discussion forum](https://www.raspberrypi.org/forums/viewforum.php?f=63) when the binaries are available.
-
 ### Update the bootloader
+* From a standard Raspbian SD card boot:
 ```
-# From a standard Raspbian SD card boot
 sudo apt update
 sudo apt full-upgrade
+```
 
-# As root edit /etc/default/rpi-eeprom-update and select BETA releases
+* As root, edit `/etc/default/rpi-eeprom-update` and select BETA releases.
 
-# Install the BETA version of the bootloader and replace the current configuration settings to enable USB boot.
-# See BOOT_ORDER property if you wish to migrate the configuration by hand.
+* Install the BETA version of the bootloader and replace the current configuration settings to enable USB boot.
+See `BOOT_ORDER` property if you wish to migrate the configuration by hand.
```

Comment:
Does "if you wish to migrate the configuration by hand" need more detailed explanation? Or do we think that everyone that's experimenting with the USB boot BETA will understand what that means?

Reply:
I was trying not to change too much of the original text. I think giving folk a link to the precise forum thread should be enough in this instance. I would rather not add additional complication to the page.